codalab/worker/bundle_service_client.py

```diff
@@ -43,6 +44,19 @@ def wrapper(*args, **kwargs):
     return decorator
 
 
+def ensure_str(response):
+    """
+    Ensure the type of response to be string
+    :param response: an response in bytes or string
+    :return: the input response in string type
+    """
+    try:
```

Comment:
Rewrite: put `if isinstance(response, str): return response` as the first line.  Then just return `response.decode()` inside the `try`.